ohcount4j

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-java -Xmx1g -cp bin:lib/* com.blackducksoftware.ohcount4j.Ohcount "$@"
```

Comment:
ohcount4j was not working as we don't have lib/* now, made changes with the help of gradle-onejar for this.